web/src/lib/cms/previews/NavBarPreview.tsx

```diff
@@ -90,7 +90,7 @@ const NavBarPreview = (props: PreviewProps): ReactElement => {
             userData={userData}
             CMS_PREVIEW_ONLY_SHOW_MENU={true}
           />
-          <div className="padding-y-10" />
+          <div className="height-card" />
```

Comment:
Why did this one and only this one divider change?

Reply:
I've updated this to allow additional spacing in the preview.